thresher/admin.py

```diff
@@ -1,6 +1,31 @@
 from django.contrib import admin
-from thresher.models import *
+from django.contrib.auth.admin import UserAdmin
+from thresher.models import (Client, Article, AnalysisType,
+							 TUA, Topic, Question, Answer, 
+							 HighlightGroup, MCSubmittedAnswer,
+							 CLSubmittedAnswer, TBSubmittedAnswer, 
+							 DTSubmittedAnswer)
 
+class UserProfileInLine(admin.StackedInline):
+	""" Class meant to serve as an inline in the 
+	newly defined user. An inline allows the 
+	UserProfile class to be modified directly 
+	through the User class.	"""
```

Comment:
Ah, I think I understand now.  On the admin page, an inline gets edited on the same page as its parent?  If so, it would be good to describe that here.  Otherwise, another description that explains the truth :)